mod/homepage/vendor/CountUp/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
now this is not a submodule.

You need to make it a submodule with the correct directory name.  you can control the directory name. read about the `git submodule` command.